packages/cli/package.json

```diff
@@ -69,7 +71,7 @@
     "@types/debug": "^4.1.7",
     "@types/ejs": "^3.1.1",
     "@types/glob": "^7.2.0",
-    "@types/node": "^18.15.11",
+    "@types/node": " 20.12.12",
```

Comment:
why the leading space? and why the version bump to 20 if we're still using 18?

Reply:
I was having some trouble getting `fetch` to resolve, so tried upgrading node types. That seemed to work, but likely only because it forced TS to reload a cache or something. Reverting this.